scripts/build-linux-kernel.sh

```
@@ -0,0 +1,79 @@
+
+# Command line
+CONFIG_CMDLINE="console=ttyAMA0 earlycon=pl011,0x09000000 loglevel=8"
+CONFIG_CMDLINE_FORCE=y
+EOF
+
+# 設定を更新
+make ARCH=arm64 CROSS_COMPILE=aarch64-linux-gnu- olddefconfig
+
+# ビルド
+echo "Building kernel..."
+make ARCH=arm64 CROSS_COMPILE=aarch64-linux-gnu- Image -j$(nproc)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)